src/VuetifyUpload.vue

```diff
@@ -59,14 +59,17 @@ export default {
     },
     computed: {
         canUploadNewFile() {
-            let count = 0;
-            this.files.forEach((file) => {
-                if (typeof file.isDeleted === 'undefined' || file.isDeleted === false) {
-                    count++;
-                }
-            });
-            return count < this.max;
+            return this.activeFiles.length < this.max;
         },
+        /**
+         * Files that need to be uploaded and files that are already uploaded and don't have the isDeleted prop
+         * the isDeleted prop is for files that are awaited deletion
+         */
+        activeFiles() {
+            return this.files.filter(file => {
+                return file.isDeleted === false || typeof file.isDeleted === 'undefined';
+            })
```

Comment:
Why make it so hard?

`return this.files.filter(file => !file.isDeleted)`

Reply:
Just copied it..